Objects/unionobject.c

```diff
@@ -300,12 +300,12 @@ union_repr(PyObject *self)
     }
 
 #if 0
-    PyUnicodeWriter_WriteUTF8(writer, "|args=", 6);
+    PyUnicodeWriter_WriteASCII(writer, "|args=", 6);
```

Comment:
For follow-up, but even if it's guarded by `if 0` for debugging purposes, shouldn't we also handle possible errors?

Reply:
Maybe. I prefer to leave the code as it is. As you wrote, it's for debugging purpose and it's disabled.